tests/test_official.py

```diff
@@ -105,9 +105,8 @@ def check_object(h4):
         field = parameter[0]
         if field == 'from':
             field = 'from_user'
-        elif name.startswith('InlineQueryResult') and field == 'type':
-            continue
-        elif name.startswith('InputMedia') and field == 'type':
+        elif (name.startswith('InlineQueryResult') or name.startswith('InputMedia')) \
```

Comment:
Could you wrap it all in paranthesis instead of using the backslash here maybe?, doesn't look very clean IMO

Reply:
👍